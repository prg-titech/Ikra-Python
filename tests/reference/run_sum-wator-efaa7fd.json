{
    "basename": "run_sum-wator-efaa7fd",
    "cmd": "new-src/interface.py --run {infile}",
    "infile": "examples/wator/wator.py",
    "infile_hash": "881d065157c07b9b943434cefd615669afe6d54e10eaf935cd8bbc61",
    "stdout": "run_sum-wator-efaa7fd.stdout",
    "stdout_hash": "c34b38a8a21e510bcd63085737c3ae5283df68e6b1bf4bdadc92ea6d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}