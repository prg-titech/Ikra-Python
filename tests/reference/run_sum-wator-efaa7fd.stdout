pygame 2.0.1 (SDL 2.0.14, Python 3.7.5)
Hello from the pygame community. https://www.pygame.org/contribute.html
7983
